pmpsui/pmps.py

```diff
@@ -194,19 +235,36 @@ def setup_tabs(self):
         self.setUpdatesEnabled(False)
 
         self.setup_fastfaults()
+        # TODO: maybe figure out how to give real progress quantities?
+        self.update_splash_message('Fast faults added', progress=40)
+
         self.setup_preemptive_requests()
+        self.update_splash_message('Preemptive requests added', progress=55)
```

Comment:
Qt bug, perhaps? The progress randomly doesn't update on this step
![image](https://github.com/pcdshub/pmps-ui/assets/10647860/9f289cfc-e76d-423a-af0f-48354dd16e30)

Reply:
A good catch.  It appears process events doesn't always trigger the redraw event.  Why this only causes issues for this step is beyond me (the progress bar was updating late very consistently here)

Reply:
It seems like it redraws the text but not the progress bar, not something to get worked up about but one of those confusing things

Reply:
The most recent commit fixed this for me, does it still not redraw the progress bar at 55% for you?